packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -105,11 +107,17 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     onCreateRemoteStreamView,
     onDisposeRemoteStreamView,
     styles,
+    layout,
     onRenderAvatar
   } = props;
 
+  let localVideoTileStyles: VideoTileStylesProps = videoTileStyle;
+  if (layout === 'floatingLocalVideo' && remoteParticipants && remoteParticipants.length > 0) {
```

Comment:
This non-trivial condition is checked more than once. Please extract a `shouldFloatLocalVideo()` helper function.

Reply:
Sure.